tests/Activities.UnitTests/Internal/ActivityMetricsSenderTests.cs

```diff
@@ -17,7 +19,70 @@ namespace Microsoft.Omex.Extensions.Activities.UnitTests.Internal
 	public class ActivityMetricsSenderTests
 	{
 		[TestMethod]
-		public void SendActivityMetric_ProduceMetrics()
+		[DataRow("")]
+		[DataRow(null!)]
+		public void ActivityMetricsSender_Construction_NonExistConfigSection_ReturnNotNull(string nonExistValue)
+		{
+			Mock<IExecutionContext> contextMock = new();
+			Mock<IHostEnvironment> environmentMock = new();
+			Mock<IConfiguration> mockConfig = new();
+			Mock<IConfigurationSection> mockConfigSection = new();
+
+			mockConfig.Setup(configuration => configuration.GetSection(ActivityMetricsSender.UseHistogramForActivityMonitoringConfigurationPath))
+				.Returns(mockConfigSection.Object);
+			mockConfigSection.Setup(section => section.Value).Returns(nonExistValue);
+
+			ActivityMetricsSender sender = new(contextMock.Object, environmentMock.Object, mockConfig.Object);
+			Assert.IsNotNull(sender);
```

Comment:
nit: maybe add a line above the `Assert`, to make it clear that the `Assert` is the final part of the *Arrange-Act-Assert* test pattern.